src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -35,36 +35,36 @@
 
 @IntegrationTest
 class AuthorTest {
-    private static BookService bookService;
+    private final BookService bookService;
 
     private static Book testBook1;
     private static Book testBook2;
-    private static AuthorService authorService;
+    private final AuthorService authorService;
     private static PredefinedShelf toRead;
 
-    @BeforeAll
-    public static void setup(@Autowired PredefinedShelfService predefinedShelfService,
-                             @Autowired BookService bookService,
-                             @Autowired AuthorService authorService) {
+    public AuthorTest(final @Autowired PredefinedShelfService predefinedShelfService,
+                      final @Autowired BookService bookService,
+                      final @Autowired AuthorService authorService) {
         toRead = new PredefinedShelfUtils(predefinedShelfService).findToReadShelf();
         testBook1 = createBook("How the mind works", toRead);
         testBook2 = createBook("The better angels of our nature", toRead);
 
-        AuthorTest.bookService = bookService;
-        AuthorTest.authorService = authorService;
+        this.bookService = bookService;
+        this.authorService = authorService;
 
-        resetBookService();
+        setUp();
```

Comment:
This can be removed

Reply:
All changes done and pushed! You can check them now